docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,91 @@ up OpenAI authentication to run the code below.
             model_type="question-answering",
         )
 
+.. _llm-eval-model-endpoint:
+
+Evaluating with a MLflow Deployment Endpoint
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For MLflow >= 2.11.0, :py:func:`mlflow.evaluate()` supports evaluating a model endpoint by directly passing the MLflow Deployment endpoint URI to the ``model`` argument.
+This is particularly useful when you want to evaluate a deployed model hosted by a local `MLflow Deployment Server <../deployments/index.html>`_ or `Databricks Foundation Model APIs <https://docs.databricks.com/en/machine-learning/model-serving/score-foundation-models.html#language-MLflow%C2%A0Deployments%C2%A0SDK>`_, without implementing custom prediction logic to wrap it as an MLflow model or a python function.
+
+Please don't forget to set the target deployment client by using :py:func:`mlflow.deployments.set_deployments_target` before calling :py:func:`mlflow.evaluate()` with the endpoint URI, as shown in the example below. Otherwise, you will see an error message like ``MlflowException: No deployments target has been set...``.
+
+.. hint::
+
+    When you want to use an external endpoint **not** hosted by a MLflow Deployment Server or Databricks, you can create a custom Python function following the :ref:`Evaluating with a Custom Function <llm-eval-custom-function>` guide and use it as the ``model`` argument.
+
+Passing Inference Parameters
+****************************
+
+You can pass additional inference parameters such as ``max_tokens``, ``temperature``, ``n``, to the model endpoint by setting the ``inference_params`` argument in :py:func:`mlflow.evaluate()`. The ``inference_params`` argument is a dictionary that contains the parameters to be passed to the model endpoint. The specified parameters are used for all the input record in the evaluation dataset.
+
+Examples
+********
+
+**Chat Endpoint hosted by a local** `MLflow Deployment Server <../deployments/index.html>`_
+
+.. code-block:: python
+
+    from mlflow.deployments import set_deployments_target
```

Comment:
Can we also import `pandas` and `mlflow` so users can copy & paste & execute this code quickly?

Reply:
great idea!